packages/@react-aria/dnd/src/utils.ts

```diff
@@ -18,6 +18,8 @@ import {Key, RefObject} from 'react';
 import {useId} from '@react-aria/utils';
 
 const droppableCollectionIds = new WeakMap<DroppableCollectionState, string>();
+export const directorySymbol = Symbol();
```

Comment:
We haven't really done something like this before, but do we want to use upper case for constants? I think that's a fairly common convention. Something like `DIRECTORY_DRAG_TYPE`? If not, I still think naming it with drag type in the name rather than "symbol" makes sense.